### ISSUE TITLE
Can't build for android after version 2.1.0

### ISSUE THREAD

**Issue by alejandromagnorsky** [OneSignal/OneSignal-Cordova-SDK#193]
Until version 2.0.11 I can run `cordova build android` without issues.

However, If I use version 2.1.0+ I get this message:
```
Execution failed for task ':transformClassesWithDexForArmv7Debug'.
> com.android.build.api.transform.TransformException: com.android.ide.common.process.ProcessException: java.util.concurrent.ExecutionException: com.android.dex.DexException: Multiple dex files define Lcom/google/android/gms/iid/zzc;
```
I believe it has something to do with using onesignal with other plugins. I tried removing and adding, and after a few tries, I found that onesignal-cordova-plugin (v2.1.0+) and cordova-plugin-google-analytics (latest version).

I'm using Cordova v7.0.1, Cordova Android v6.2.3 and Node v7.10.0.

**Comment by jkasten2** [comment]
@alejandromagnorsky Sorry about the issue. Just released a 2.1.2 update to fix this problem.

Thanks.

**Comment by alejandromagnorsky** [comment]
Wow, that was really a "quick fix". it's working now. Thanks!
